src/simulation.py

```diff
@@ -81,8 +83,11 @@ def reset(self):
         self.population = self.new_gen()
         self.screen = screen_init(self.config)
     
+    
     def evolve(self):
         self.running = True
         while self.running:
+            self.generation += 1
+            print("Generation:", self.generation)
```

Comment:
TODO: Counter get's displayed in pygame
```suggestion
            print("Generation:", self.generation_counter)
```